test/events.js

```diff
@@ -1,4 +1,9 @@
+const annotationTypes = ['box', 'ellipse', 'label', 'line', 'point', 'polygon'];
+
 export function testEvents(options, innerElement) {
+  if (!annotationTypes.includes(options.type)) {
+    console.warn(`No annotation found with type '${options.type}'.`);

```

Comment:
I was thinking the warning should be output when creating the annotation elements. I'll create a pr for that.

Reply:
Created #530

Reply:
Yes, I did before that PR. I'm going to remove it

Reply:
And also merge because a rebase is needed

Reply:
fixed and rebased